core/embed/upymod/modtrezormintlayer/utils.c

```diff
@@ -9,40 +9,40 @@ void handle_err(ByteArray *res) {
 
   switch (res->len_or_err.err) {
     case WrongHashSize:
-      mp_raise_ValueError("Invalid hash size");
+      mp_raise_ValueError(MP_ERROR_TEXT("Invalid hash size"));
```

Comment:
FYI: these changes fix compilation after the merge from upstream. Technically they should be part of https://github.com/mintlayer/mintlayer-trezor-firmware/pull/12 but I decided to put them here for easier review.